packages/terra-time-input/src/TimeUtil.js

```diff
@@ -34,4 +170,44 @@ class TimeUtil {
   }
 }
 
+TimeUtil.validateTime = new RegExp('^(0[0-9]|1[0-9]|2[0-3]):[0-5][0-9]$');
+
+TimeUtil.inputType = {
+  HOUR: 0,
+  MINUTE: 1,
+};
+
+TimeUtil.keyCodes = {
+  BACKSPACE: 8,
+  ARROWLEFT: 37,
+  ARROWUP: 38,
+  ARROWRIGHT: 39,
+  ARROWDOWN: 40,
+  DELETE: 46,
+};
+
+TimeUtil.isConsideredMobileDevice = () =>
+  window.matchMedia('(max-width: 1024px)').matches &&
+  (
+    'ontouchstart' in window ||
+    // eslint-disable-next-line no-undef
+    (window.DocumentTouch && document instanceof DocumentTouch) ||
+    navigator.maxTouchPoints > 0 ||
+    navigator.msMaxTouchPoints > 0
+  );
+
+Object.defineProperty(TimeUtil, 'FORMAT_12_HOUR', {
+  value: '12-hour',
+  configurable: false,
```

Comment:
Could we document why we define configurable, enumerable, writable? They are the same for '12-hour' & '24-hour'

Reply:
https://github.com/cerner/terra-core/pull/1027/commits/8d993b43e0232da22d729afb55fc6225dd22b2e6